CHANGELOG.asciidoc

```diff
@@ -19,6 +19,9 @@ https://github.com/elastic/beats/compare/v5.4.1...master[Check the HEAD diff]
 *Heartbeat*
 
 *Metricbeat*
+- Change `system.process.cpu.total.pct` to be normalized by the number of CPU cores
+  such that the value will now range between 0 and 100% which is consistent
+  with CPU usage percentages provided by the system cpu metricset.
```

Comment:
Can you add the PR number? Also, I would optimistically add it under the 5.5.0 section, the final BC is not yet started, I think.